doc/manuals/admin/database_model.md

```diff
@@ -86,55 +86,55 @@ Fields:
 Example document:
 
 ```
- {
```

Comment:
JSON hasn't changed actually, but for some reason github shows this as change...

NTC